libiocage/lib/Jail.py

```diff
@@ -314,6 +317,24 @@ def basejail_backend(self):
 
         return None
 
+    def _run_hook(self, hook_name: str):
+
+        key = f"exec_{hook_name}"
+        value = self.config[key]
+
+        if value == "/usr/bin/true":
+            return
+
+        self.logger.verbose(
+            f"Running {hook_name} hook for {self.humanreadable_name}"
+        )
+
+        return libiocage.lib.helpers.exec(
+            value.split(" "),
```

Comment:
So if we split on white space, you will lose the exec_start and exec_stop and such. Since it's /bin/sh/etc/rc*

Reply:
We need to set the subprocess argument shell=True or split the command and it's arguments into a list.

Reply:
We shouldn't shell with subprocess. That's just another can of worms :D We should yes make it a list (why I marked this as needs changes) or we can use shlex to parse it into the list too.